src/game/PlayerBots/PlayerBotMgr.cpp

```diff
@@ -1763,6 +1762,28 @@ bool ChatHandler::HandleBattleBotRemoveAllCommand(char* args)
     return true;
 }
 
+bool ChatHandler::HandleBattleBotAutoJoinCommand(char* args)
+{
+    if (!*args)
+    {
+        SendSysMessage(m_confBattleBotAutoJoin ? "Battlebots AutoJoin - on" : "Battlebots AutoJoin - off");
```

Comment:
```
/home/runner/work/core/core/src/game/PlayerBots/PlayerBotMgr.cpp: In member function ‘bool ChatHandler::HandleBattleBotAutoJoinCommand(char*)’:
/home/runner/work/core/core/src/game/PlayerBots/PlayerBotMgr.cpp:1769:24: error: ‘m_confBattleBotAutoJoin’ was not declared in this scope
 1769 |         SendSysMessage(m_confBattleBotAutoJoin ? "Battlebots AutoJoin - on" : "Battlebots AutoJoin - off");
      |                        ^~~~~~~~~~~~~~~~~~~~~~~
/home/runner/work/core/core/src/game/PlayerBots/PlayerBotMgr.cpp:1782:5: error: ‘m_confBattleBotAutoJoin’ was not declared in this scope
 1782 |     m_confBattleBotAutoJoin = value ? true : false;
      |     ^~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/game/CMakeFiles/game.dir/build.make:2925: src/game/CMakeFiles/game.dir/PlayerBots/PlayerBotMgr.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:524: src/game/CMakeFiles/game.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```
`m_confBattleBotAutoJoin` is a member variable of `PlayerBotMgr` but `ChatHandler::HandleBattleBotAutoJoinCommand` is a member function of `ChatHandler` so you can't access it here.